libultraship/libultraship/InputEditor.cpp

```diff
@@ -172,18 +170,21 @@ namespace Ship {
 				DrawButton("Right", BTN_VSTICKRIGHT);
 
 				ImGui::SetCursorPosX(ImGui::GetCursorPosX() + 8);
-				DrawVirtualStick("##CameraVirtualStick", ImVec2(Backend->wCamX / sensitivity, Backend->wCamY / sensitivity));
+				// 2 is the SDL value for right stick X axis
+				// 3 is the SDL value for right stick Y axis.
+				DrawVirtualStick("##CameraVirtualStick", ImVec2(Backend->getRightStickX(CurrentPort) / profile->AxisSensitivities[2], Backend->getRightStickY(CurrentPort) / profile->AxisSensitivities[2]));
 
 				ImGui::SameLine();
 				ImGui::SetCursorPosX(ImGui::GetCursorPosX() + 5);
 				ImGui::BeginChild("##CSInput", ImVec2(90, 85), false);
 					ImGui::Text("Deadzone");
 					ImGui::PushItemWidth(80);
-					ImGui::InputFloat("##MDZone", &profile.Thresholds[RIGHT_STICK], 1.0f, 0.0f, "%.0f");
+					ImGui::InputFloat("##MDZone", &profile->AxisDeadzones[2] /* This is the SDL value for left stick X axis */, 1.0f, 0.0f, "%.0f");
```

Comment:
just want to confirm these comments match up with the values here. reading through all of them we have
|  | x | y |
| - | - | - |
| left stick | 2 | ? |
| right stick | 2 | 3 |

Reply:
Left Stick X = 0, Left Stick Y = 1
Right Stick X = 2, Right Stick Y = 3